app/models/piece.rb

```diff
@@ -23,18 +23,21 @@ def is_obstructed?(x_target, y_target)
 
 
   #determines if the move is horizontal
-  def horizontal_move?(x_target, y_target)
+  def horizontal_move?(x, y, x_target, y_target)
+    return if (x_target - x).abs > 0 && (y - y_target).abs ==  0
```

Comment:
The math is correct either way, but I think we should use `(y_target - y).abs` here to stay consistent with the format of the other two "move?" methods.

Reply:
I was def not paying attention while writing that. Easy fix!